### PR TITLE
Bot core

### PR DESCRIPTION
**Pull request by kirillta**

### CODE REVIEW

File: api/guilds.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    discord_client: commands.Bot
+
+    def __init__(self, discord_client: commands.Bot) -> None:
+        self.discord_client = discord_client
+
+    def get_members(self, guild_id: int) -> list[discord.Member]:
+        guild = self.discord_client.get_guild(guild_id)
+
+        members = []
+        for member in guild.members:
```
Comment (konstyefremov) [inline api/guilds.py:15]: The whole loop could be replaced with a single list comprehension:
members = [x for x in guild.members if x.bot is False]

File: api/guilds.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+            members.append(member)
+
+        return members
+
+    def get_online_members(self, guild_id: int) -> list[discord.Member]:
+        members = self.get_members(guild_id)
+
+        online_members = []
+        for member in members:
```
Comment (konstyefremov) [inline api/guilds.py:27]: The whole loop could be replaced with list comprehension
online_members = [x for x in members if x.status is discord.Status.online]

File: bot.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+@slash.slash(name='scripter', description='Slay a sacrifice!', guild_ids=[GUILD_ID])
+async def get_scripter(ctx):
+    try:
+        restrictd_member_ids = config['app']['restrictdMemberIds']
+        service = ScripterService(bot)
+        member = service.get(GUILD_ID, restrictd_member_ids)
+
+        message: str
+        if member is None:
```
Comment (konstyefremov) [inline bot.py:42]: Could be replaced into python ternary with a single call to ctx.send(message)

```
message = ('🔮 Astrologers announced that there is none to choose from 🔮'
           if member is None
           else f'📜 Astrologers hast chosen {member.mention}')
```

File: bot.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+@slash.slash(name='ping', description='Test', guild_ids=[GUILD_ID])
+async def ping(ctx):
+    await ctx.send('pong')
+
+
+@slash.slash(name='scripter', description='Slay a sacrifice!', guild_ids=[GUILD_ID])
+async def get_scripter(ctx):
+    try:
+        restricted_member_ids = config['app']['restrictdMemberIds']
```
[the remark this answers is not in the document]
Comment (kirillta) [reply bot.py:37]: No, still typos

**Review by konstyefremov — APPROVED** [review]

**Pull request merged**